diff-pr.sh

```diff
@@ -115,7 +115,7 @@ template='
 	{{- "\n" -}}
 	{{- range $.Entries -}}
 		{{- $arch := .HasArchitecture arch | ternary arch (.Architectures | first) -}}
-		{{- $from := $.ArchDockerFrom $arch . -}}
+		{{- $froms := $.ArchDockerFroms $arch . -}}
```

Comment:
This one has me wondering if we should add an explicit `.EnsureGitFetched` function, since we use this function in a lot of places to mean that. :sweat_smile:

(Room for future improvement -- I don't think we should block this on that.)